utils/utils.go

```diff
@@ -0,0 +1,40 @@
+package utils
```

Comment:
If these functions are only supposed to be used with tests then you should either rename the file `utils_test.go` or make a package for testing functions because they are only valid in the testing usecases and will confuse other developers.